Drive/rover_drive_system.hpp

```diff
@@ -339,6 +339,18 @@ class RoverDriveSystem : public sjsu::common::RoverSystem
       throw e;
     }
   }
+  double calculateLeftTurnAngle(double x_angle)
+  {
+        double left_angle = static_cast<double>(0.392 + 0.744 * x_angle + -0.0187 * pow(x_angle, 2) +
+        1.84E-04 * pow(x_angle, 3));
+        return left_angle;
+  }
+  double calculateBackTurnAngle(double x_angle)
+  {
+        double back_angle = static_cast<double>(-0.378 + -1.79 * x_angle + 0.0366 * pow(x_angle, 2) +
+        -3.24E-04 * pow(x_angle, 3));
+        return back_angle;
```

Comment:
```suggestion
        return (static_cast<double>(-0.378 + -1.79 * x_angle + 0.0366 * pow(x_angle, 2) +
        -3.24E-04 * pow(x_angle, 3)));
```
possibly refactor, not necessary...